components/Posts.php

```diff
@@ -182,6 +191,7 @@ protected function listPosts()
             'category'   => $category,
             'published'  => $isPublished,
             'exceptPost' => $this->property('exceptPost'),
+            'exceptCategories' => $this->property('exceptCategory'),
```

Comment:
Please align the indentation of the `=>` from the above properties to this once since it's now the longest.